SPARCoreTests/SPARCoreTests.m

```diff
@@ -43,7 +41,7 @@ - (void)testJsonParser {
     XCTAssertEqualObjects(expectedArticle.articleId, returnValues[0].articleId);
     XCTAssertEqualObjects(expectedArticle.title, returnValues[0].title);
 }
-
+*/
```

Comment:
I've reviewed it. Since this test is testing the ArticleFromJSON method, but now this method is already replaced by ArticleFromArray, thus I cannot test it... or I should test ArticleFromArray instead?